extensions/wikia/AssetsManager/config.php

```diff
@@ -215,6 +215,7 @@
 		'//extensions/wikia/AdEngine/js/context/adContext.js',
 		'//extensions/wikia/AdEngine/js/lookup/lookupFactory.js',
 		'//extensions/wikia/AdEngine/js/slot/adSlot.js',
+		'//extensions/wikia/AdEngine/js/slot/adUnitBuilder.js',
```

Comment:
It's needed by `//extensions/wikia/ARecoveryEngine/js/helper.js` and right now we need it for mobile too :(